tomcat/src/main/java/com/techcourse/controller/RegisterController.java

```diff
@@ -16,25 +16,26 @@ public RegisterController(UserService userService) {
     }
 
     @Override
-    protected HttpResponse doGet(HttpRequest request) {
-        return HttpResponse.redirect("/register.html").build();
+    protected void doGet(HttpRequest request, HttpResponse response) {
+        response.setRedirect("/register.html");
     }
 
     @Override
-    protected HttpResponse doPost(HttpRequest request) {
+    protected void doPost(HttpRequest request, HttpResponse response) {
         RequestBody requestBody = request.getRequestBody();
 
         String account = requestBody.get("account");
```

Comment:
```suggestion
        String account = request.getParameter("account");
```

컨트롤러가 RequestBody를 알아야 할 필요가 있을까요?

Reply:
좋은 지적인것 같습니다!
requestBody를 의존하지 않도록 수정하겠습니다!